src/test/java/org/broadinstitute/hellbender/tools/spark/sv/RunSGAViaProcessBuilderOnSparkUnitTest.java

```diff
@@ -95,12 +96,40 @@ public void writeToLocalTest() throws IOException{
         Assert.assertTrue( FileUtils.contentEqualsIgnoreEOL(copied._2(), originalFASTQFile, StandardCharsets.UTF_8.name()) );
     }
 
+    // Test if disabling stdout stderr works
+    @Test(groups = "sv")
+    public void stdioCaptureTest() throws IOException{
```

Comment:
I did try that once, but the process spawned don't produce stderr (normally, unless something went wrong). So I skipped that half of the test. On second thought, I could deliberately let the program fail (e.g. providing wrong arguments, ask to log), and see if stdout and stderr is captured.
Thanks!